st2common/st2common/rbac/types.py

```diff
@@ -38,8 +38,8 @@ class PermissionType(Enum):
     PACK_ALL = 'pack_all'
 
     # Note: Right now we only have read endpoints for sensors types
-    SENSOR_VIEW = 'sensor_view'
-    SENSOR_ALL = 'sensor_all'
+    SENSOR_VIEW = 'sensor_type_view'
```

Comment:
I explicitly decided to use sensor instead of sensor type to hide the type abstraction here since I don't think it's necessary.